### PR TITLE
scripts: replace com.termux with @TERMUX_APP_PACKAGE@

### PR DESCRIPTION
**Pull request by t184256**
I believe that this variable isn't used consistently across scripts/.

"Draft" because I'd like to test it more thoroughly.

### CODE REVIEW

File: scripts/termux-open.in [hunk 1]
```diff
@@ -44,7 +44,7 @@ case "${TERMUX__USER_ID:-}" in ''|*[!0-9]*|0[0-9]*) TERMUX__USER_ID=0;; esac
 
 am broadcast --user "$TERMUX__USER_ID" \
 	-a $ACTION \
-	-n com.termux/com.termux.app.TermuxOpenReceiver \
+	-n @TERMUX_APP_PACKAGE@/@TERMUX_APP_PACKAGE@.TermuxOpenReceiver \
```
Comment (agnostic-apollo) [inline scripts/termux-open.in:47]: `.app` missing.

**Comment by agnostic-apollo** [comment]
I already noticed this locally, but termux-tools needs to use env variables that will be exported in Termux app `v0.119.0`, and build time placeholders only need to be used as fallback. Also `TERMUX_` root scope may be different so scripts being added need to be used instead. Same for other placeholders. But this can be merged for now for initial fix.

**Comment by agnostic-apollo** [comment]
Also `TERMUX_APP_PACKAGE` is for the app package name, but forks may be using a different class package name than `com.termux.app`, so separate variable needs to be added in `properties.sh` and env for it.

**Comment by t184256** [comment]
I'm not sure I understood it right; like this?

**Comment by t184256** [comment]
(turns out I do have a fork suffixing app package name, but not the class names)

**Comment by agnostic-apollo** [comment]
No no, just use `@TERMUX_APP_PACKAGE@.app` for now. Class package name support needs to be added in termux-packages repo, I'll do it later.

**Comment by t184256** [comment]
OK, reverted.

**Comment by agnostic-apollo** [comment]
Had to fix `termux-reset` for changes made [previously](https://github.com/termux/termux-tools/blob/acf2a206e7aa22b3d4e4d8f6d6bf7d7565381025/scripts/termux-reset.in#L48):
- The `/.app.TermuxService` had to be replaced with `/@TERMUX_APP_PACKAGE@.app`, otherwise it would have expanded to fork app package name instead of termux classes package name. Even that is not technically correct as termux package name may not be the same as its namespace in a fork.
- The `com.termux.termuxam.Am` must not be replaced with `@TERMUX_CLASS_NAME@.termuxam.Am`, as that class does not belong in termux app, but in a different repo and has a different namespace/package name than `com.termux`. It should only be changed if some fork moves the classes to a different package name.
    - https://github.com/termux/TermuxAm/blob/v0.8.0/app/src/main/java/com/termux/termuxam/Am.java
    - https://developer.android.com/build/configure-app-module#set-namespace
    
Additional fixes will be done later, read the commit message https://github.com/termux/termux-tools/pull/95/commits/e50eb264f648bd6b09ddbedb0cecdf93cb4de643

Whoever updates the termux-tools package, should either ping me or check the paths themselves in the built deb as I haven't tested.

**Pull request merged**